subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -1158,14 +1160,10 @@ static void isr_done(void *param)
 	}
 #endif /* CONFIG_BT_CTLR_ADV_INDICATION */
 
-#if defined(CONFIG_BT_CTLR_ADV_EXT)
-	struct event_done_extra *extra;
-
-	extra = ull_event_done_extra_get();
-	LL_ASSERT(extra);
-
-	extra->type = EVENT_DONE_EXTRA_TYPE_ADV;
-#endif  /* CONFIG_BT_CTLR_ADV_EXT */
+#if defined(CONFIG_BT_CTLR_ADV_EXT) || \
+	defined(CONFIG_BT_CTLR_JIT_SCHEDULING)
+	ull_set_done_extra_type(EVENT_DONE_EXTRA_TYPE_ADV);
```

Comment:
```suggestion
	ull_event_done_extra_type_set(EVENT_DONE_EXTRA_TYPE_ADV);
```

Side note, may be in a separate PR, all other places we can use this function.